kernel/kernel-impl/src/test/java/org/sakaiproject/content/impl/test/RoleAccessTest.java

```diff
@@ -66,11 +66,17 @@ public void setUp() throws IdUsedException, IdInvalidException, InconsistentExce
         session.setUserId("admin");
 
         _ss = (SiteService)getService(SiteService.class.getName());
-        Site newSite = _ss.addSite(SITE_ID, (String) null);
-        Group group = newSite.addGroup();
-        group.setTitle(".group");
-        _groupReference = group.getReference();
-        _ss.save(newSite);
+        Site newSite = null;
```

Comment:
Anyway, I think for these tests it doesn't matter having a new site with one group each time or re-using it, as long as the collections and roles are modified.